### ISSUE TITLE
Add configuration file

### ISSUE THREAD

**Issue by Limerio** [BuckshotPlusPlus/BuckshotPlusPlus#51]
## Configuration in file
- port (dev or prod ***SSR Only***)
- public folder (images, json, etc)
- multi format yml, json, cfg (if it's possible, it's help the community to have this possibility)

**Comment by Vic92548** [comment]
Good idea, I waas thinking of maybe .env for that, it's simple, easy to use, widely used now and useful to not push sensible config, like API keys or database passwords
